modules/payments/modules/payments/README.md

```diff
@@ -1,59 +1,87 @@
 
 
-# iOS
-making sure the minimal iOS platform is set to 11 in `ios/Podfile` file
+# Payment Module 
 
-if you see this error
-Undefined symbols for architecture x86_64 on iOS
-While building your iOS project, you may see a Undefined symbols for architecture x86_64 error. This is caused by react-native init template configuration that is not fully compatible with Swift 5.1.
+Payment module supports following functionalities
 
-Undefined symbols for architecture x86_64:
-  "(extension in Foundation):__C.NSScanner.scanUpToString(Swift.String) -> Swift.String?", referenced from:
-      static Stripe.STPPhoneNumberValidator.formattedRedactedPhoneNumber(for: Swift.String, forCountryCode: Swift.String?) -> Swift.String in libStripe.a(STPPhoneNumberValidator.o)
-  "__swift_FORCE_LOAD_$_swiftUniformTypeIdentifiers", referenced from:
-      __swift_FORCE_LOAD_$_swiftUniformTypeIdentifiers_$_Stripe in libStripe.a(PKPaymentAuthorizationViewController+Stripe_Blocks.o)
-Follow these steps to resolve this:
+- Stripe Payment
+- Apple Pay (through stripe)
+- Google Pay (through stripe)
+- [Apple inApp Purchase (through apple)](#headin)
 
-### Solution
-Open your project via Xcode, go to project -> build settings, find library search paths and remove all swift related entries such as: $(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME) and $(TOOLCHAIN_DIR)/usr/lib/swift-5.0/$(PLATFORM_NAME).
-Create a new Swift file to the project (File > New > File > Swift), give it any name (e.g. Fix.swift) and create a bridging header when prompted by Xcode.
+# Setup iOS
+Follow following steps to setup the project in iOS.
 
+- NOTE: Before doing` pod install`
+- open `ios/Podfile` file and set `platform :ios, '11.0'` if its 10 or less than 11.
+- Then do `pod install`
+- open the `ios/PROJECT_NAME.xcworkspace` file in Xcode.
+- Create a new Swift file to the project `File > New > File > Swift File`, give it any name (e.g. Fix.swift) and create a `bridging header` when prompted by Xcode.
+- select the project and click on `build settings` search for `Library Search Paths` double click values and remove all swift related entries such as: \$(TOOLCHAIN_DIR)/usr/lib/swift/\$(PLATFORM_NAME) and \$(TOOLCHAIN_DIR)/usr/lib/swift-5.0/\$(PLATFORM_NAME)
 
-# Android
-AndroidManifest.xml
-<application>
-...
-<meta-data
-        android:name="com.google.android.gms.wallet.api.enabled"
-        android:value="true" />
-...
-</application>
 
+### **Setup Apple inAPP purchase Local testing environment**
```

Comment:
```suggestion
### **Setup Apple In-App Purchase Local testing environment**
```
Readability suggestion. Name it the same way as Apple does:
https://developer.apple.com/in-app-purchase/